migration-engine/connectors/migration-connector/src/database_migration_inferrer.rs

```diff
@@ -9,6 +9,9 @@ pub trait DatabaseMigrationInferrer<T>: Send + Sync {
     /// Infer the database migration steps. The previous datamodel is provided, but the implementor can ignore it.
     async fn infer(&self, previous: &Datamodel, next: &Datamodel, steps: &[MigrationStep]) -> ConnectorResult<T>;
 
+    /// Infer the database migration steps, skipping the schema describer and assuming an empty database.
```

Comment:
Since this is part of the connector API, we should formulate the docs in a connector-agnostic way. Suggestion:

"     /// Infer the database migration steps assuming an empty schema on a new database as a starting point."

Reply:
Fixed in https://github.com/prisma/prisma-engines/pull/1108/commits/b6d384652d21f6f394527c4438fe335835e8a679